Cargo.toml

```diff
@@ -25,6 +25,7 @@ __test = []
 crossbeam-utils = { version = "0.8.12", default-features = false }
 parking = { version = "2.0.0", optional = true }
 slab = { version = "0.4.7", default-features = false }
+spin = { version = "0.9.4", default-features = false, features = ["spin_mutex"] }
```

Comment:
imo it would be more readable to replace this with
```toml
[dependencies.spin]
version = "0.9.4"
default-features = false
features = ["spin_mutex"]
```